integration/deploy/manifest_test.go

```diff
@@ -42,13 +42,70 @@ var (
     context: app
 deploy:
   - kubectl apply -f k8s.yml
+`
+	oktetoManifestContentWithCache = `build:
+  app:
+    context: app
+	export-image: okteto.dev/app:dev
```

Comment:
this should have the same indentation than `context`, no?

Reply:
Yes, I used tabs instead of whitespaces and that's why it didn't have the same indentation but for Vscode it was